src/app/components/url-box/url-box.component.ts

```diff
@@ -9,12 +10,13 @@ export class UrlBoxComponent {
   @Input() httpVerb: string;
   @Input() isSubscribed = false;
   @Input() isLoading = false;
+  @Input() isMac = navigator.appVersion.includes('Mac');
   @Input() showDocs = false;
   @Input() selectedOperation = '';
   @Input() queryOperations = [];
   @Input() streamState = '';
   @Input() currentCollection = null;
-
+  @Input() sendRequestKeybinds = SEND_REQUEST_KEYBINDS;
```

Comment:
You aren't using this as an input, but you should. The url-box component can remain a dumb component with no access to any service. The `SEND_REQUEST_KEYBINDS` can be passed as an input from the window component instead: https://github.com/imolorhe/altair/blob/staging/src/app/containers/window/window.component.html#L15-L23